experiments/twofsound.py

```diff
@@ -3,112 +3,50 @@
-            "pulse_phase_offset": [0, 0, 0, 0, 0, 0, 0, 0],
-            "tau_spacing": tau_spacing,
-            "pulse_len": 300,  # us
-            "num_ranges": 75,  # range gates
-            "first_range": 180,  # first range gate, in km
+            "pulse_sequence": scf.SEQUENCE_7P,
+            "tau_spacing": scf.TAU_SPACING_7P,
+            "pulse_len": scf.PULSE_LEN_45KM,
+            "num_ranges": scf.STD_NUM_RANGES,
+            "first_range": scf.STD_FIRST_RANGE,
             "intt": 3500,  # duration of an integration, in ms
-            "beam_angle": [-26.25, -22.75, -19.25, -15.75, -12.25, -8.75,
-                           -5.25, -1.75, 1.75, 5.25, 8.75, 12.25, 15.75, 19.25, 22.75,
-                           26.25],
-            "beam_order": [15, 14, 13, 12, 11, 10, 9, 8, 7, 6, 5, 4, 3, 2, 1, 0],
-            #"scanboundflag": True,  # there is a scan boundary
-            #"scanbound": 60000,  # ms
-            #"clrfrqflag": True,  # search for clear frequency before transmitting
-            #"clrfrqrange": [13100, 13400],  # frequency range for clear frequency search,
-            "txfreq" : 10500,
-            # kHz including a clrfrqrange overrides rxfreq and txfreq so these are no
-            # longer necessary as they will be set by the frequency chosen from the
-            # range.
+            "beam_angle": scf.STD_16_BEAM_ANGLE,
+            "beam_order": scf.STD_16_REVERSE_BEAM_ORDER,
+            "txfreq" : 10500, #kHz
             "acf": True,
             "xcf": True,  # cross-correlation processing
             "acfint": True,  # interferometer acfs
         }
 
-        slice_2 = {  # slice_id = 1
-            "tx_antennas": tx_ant,
-            "rx_main_antennas": rx_main_ant,
-            "rx_int_antennas": rx_int_ant,
-            "pulse_sequence": pulse_sequence,
-            "pulse_phase_offset": [0, 0, 0, 0, 0, 0, 0, 0],
-            "tau_spacing": tau_spacing,
-            "pulse_len": 300,  # us
-            "num_ranges": 75,  # range gates
-            "first_range": 90,  # first range gate, in km
-            "intt": 3500,  # duration of an integration, in ms
-            "beam_angle": [-26.25, -22.75, -19.25, -15.75, -12.25, -8.75,
-                           -5.25, -1.75, 1.75, 5.25, 8.75, 12.25, 15.75, 19.25, 22.75,
-                           26.25],
-            "beam_order": [15, 14, 13, 12, 11, 10, 9, 8, 7, 6, 5, 4, 3, 2, 1, 0],
-            #"scanboundflag": True,  # there is a scan boundary
-            #"scanbound": 60000,  # ms
-            #"clrfrqflag": True,  # search for clear frequency before transmitting
-            #"clrfrqrange": [10200, 10500],  # range for clear frequency search, kHz
-            "txfreq": 13000,
-            "acf": True,
-            "xcf": True,  # cross-correlation processing
-            "acfint": True,  # interferometer acfs
-        }
+        slice_2 = copy.deepcopy(slice_1)
+        slice_2['first_range'] = 90 #km
```

Comment:
90km, cool, but when did this change for the second frequency? Did Pasha suggest it? I don't know why we don't just measure from 0 all the time.

Reply:
They should probably both be at 180.

Reply:
I updated this.